static_src/components/service_instance_list.jsx

```diff
@@ -0,0 +1,102 @@
+
+import React from 'react';
+import Reactable from 'reactable';
+
+import Button from './button.jsx';
+import serviceActions from '../actions/service_actions.js';
+import ServiceInstanceStore from '../stores/service_instance_store.js';
+
+var Table = Reactable.Table,
+    Thead = Reactable.Thead,
+    Th = Reactable.Th,
+    Tr = Reactable.Tr,
+    Td = Reactable.Td;
+
+function stateSetter(props) {
+  return {
+    serviceInstances: ServiceInstanceStore.getAll(),
+    currentSpaceGuid: props.initialSpaceGuid
+  };
+}
+
+export default class ServiceInstanceList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+    this.state = stateSetter(props);
+  }
+
+  componentDidMount() {
+    ServiceInstanceStore.addChangeListener(this._onChange);
+    serviceActions.fetchAllInstances(this.state.currentSpaceGuid);
+  }
+
+  _onChange = () => {
+    this.setState(stateSetter(this.props));
+  }
+
+  _handleDelete = (instanceGuid, ev) => {
+    ev.preventDefault();
```

Comment:
Prevent default ensures that the default action for the event doesn't happen. So if you have an a tag with a link, then the default action would be for the browser to go to the link. Prevent default will stop that from happening. In general, it's used to ensure browser consistent behavior, although often it's not strictly required.